editor/src/messages/tool/tool_messages/pen_tool.rs

```diff
@@ -306,13 +312,21 @@ struct PenToolData {
 
 	previous_handle_start_pos: DVec2,
 	previous_handle_end_pos: Option<DVec2>,
+	toggle_colinear_debounce: bool,
+	colinear: bool,
 	alt_press: bool,
+	space_press: bool,
+	lock_toggle: bool,
+	handle_swap: bool,
+	angle_locked: bool,
+	close_path: bool,
```

Comment:
Should these be renamed to indicate that they are a status? "press" -> "pressed", "toggle" -> "toggled", "swap" -> "swapped", "close_path" -> "path_closed"